src/python/update_container_definitions.py

```diff
@@ -26,7 +26,7 @@ def run(previous_task_definition, container_image_name_updates, container_env_va
 
     # Expected format: container=...,name=...,value=...,container=...,name=...,value=
     try:
-        env_kv_pairs = container_env_var_updates.split(',')
+        env_kv_pairs = container_env_var_updates.replace("'", "\\'").split(',')
```

Comment:
I tried testing with task definitions containing single quotes (https://github.com/CircleCI-Public/aws-ecs-orb/commit/9f6b5686eafae70824ff9fe5b3a65921c0200332) and found that the integration tests had failed (https://circleci.com/gh/CircleCI-Public/aws-ecs-orb/2085) It looks like there might have been issues with sourcing the resultant bash script after the escaping.

Reply:
Let me dig deeper into this